frontend/package.json

```diff
@@ -87,6 +87,7 @@
     "sanitize-html": "1.x",
     "showdown": "1.8.x",
     "text-encoding": "0.x",
+    "toggle-fullscreen": "^0.3.3",
```

Comment:
This library isn't widely used or very active (0 stars), which makes me a little hesitant to add it.

I'd suggest something like https://github.com/sindresorhus/screenfull.js

Reply:
(noticed that as well and was a little concerned)

Reply:
Hi, switched to screenfull, thanks